include/rocksdb/utilities/ldb_cmd.h

```diff
@@ -61,6 +61,10 @@ class LDBCommand {
   static const std::string ARG_CREATE_IF_MISSING;
   static const std::string ARG_NO_VALUE;
   static const std::string ARG_DISABLE_CONSISTENCY_CHECKS;
+  static const std::string ARG_ENABLE_BLOB_FILES;
+  static const std::string ARG_MIN_BLOB_SIZE;
+  static const std::string ARG_BLOB_FILE_SIZE;
+  static const std::string ARG_BLOB_COMPRESSION_TYPE;
```

Comment:
The other options are about garbage collection. It's looks like something very internal and may make `ldb` complicated. I think it would be meaningful to expose when there are some use cases.

Reply:
There is a use case already: when you load data using `ldb`, automatic compactions run in the background (unless you disable them via the command line). We also have an `ldb` command called `compact` that can be used to manually compact the database (a range of keys or even the whole thing). Since BlobDB's GC is integrated with the LSM tree compaction process, it makes sense to expose these parameters to ensure the user has control over the GC process.